lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -1,36 +1,42 @@
 # Guidelines for AWS modules
 
+The Ansible AWS modules and these guidelines are maintained by the
+Ansible AWS Working Group.  For further information see
+[the AWS working group community page](https://github.com/ansible/community/tree/master/group-aws).
+If you are planning to contribute AWS modules to Ansible then getting in touch with the the working
+group will be a good way to start, especially because a similar module may already be under
+development.
+
 ## Getting Started
 
-Since Ansible 2.0, it is required that all new AWS modules are written to use boto3.
+Since Ansible 2.0, it is required that all new AWS modules are written to use boto3.  Please do not
+add new dependencies to the old boto library.
 
-Prior to 2.0, modules may have been written in boto or boto3. The effort to port all modules to boto3 has begun.
+Prior to 2.0, modules may have been written in boto or boto3. The effort to port all modules to boto3 has begun.   From Ansible 2.4 it is permissible for modules which previously required boto to start to require boto3 in order to deliver the functionality previously supported with boto.
```

Comment:
I have done reasonably aggressive wrapping all through the document so now all should be fine.